test/server/cards/02_SHD/units/TheMandalorian.spec.ts

```diff
@@ -0,0 +1,42 @@
+describe('The Mandalorian, Wherever I Go He Goes', function () {
+    integration(function (contextRef) {
+        describe('The Mandalorian\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['the-mandalorian#wherever-i-go-he-goes'],
+                        groundArena: [ 
+                            { card: 'grogu#irresistible', damage: 3 },
+                            { card: 'mandalorian-warrior' }
+                        ]
+                    }
+                })
+            })
+
+            it('should heal all damage and give 2 shield tokens to a unit', function () {
+                const { context } = contextRef
+                const promptMessage = 'You may heal all damage from a unit that costs 2 or less and give 2 Shield tokens to it'
```

Comment:
maybe you can put this const a little higher in the describe to use it on the second case

Reply:
I added an enemy unit, then it seems the prompt message wasn't appearing anymore, it was just the selection of units and the pass button. Then i adjusted the assertion to reflect this, is it ok?

Reply:
Yeah that's expected. If there are multiple targets to choose from it switches modes so that the Pass option is a button included as part of targeting. That way you don't have to click a button and then choose a target.